Acceptance_test.go

```diff
@@ -18,46 +21,211 @@ import (
+				Expect(math.Floor(runningTimeValue)).To(Equal(float64(5)))
+			})
+
+			It("List Usage", func() {
+				list := []string{
+					fmt.Sprintf(`%s -X GET'`, TestServer.CreateURL("/people")),
+				}
+				summary, err := test.ExecuteList(list, "--summary", "--wait-time", "1s", "--iterations", "5")
+				Expect(err).To(BeNil())
+
+				var runningTimeToParse = strings.Replace(summary.RunningTime, "s", "", -1)
+				runningTimeValue, _ := strconv.ParseFloat(runningTimeToParse, 64)
+				Expect(math.Floor(runningTimeValue)).To(Equal(float64(5)))
+			})
+		})
+
+		Describe("Duration", func() {
+
+			It("Plan Usage", func() {
+
+				var plan = fmt.Sprintf(`---
+name: Some Plan
+iterations: 0
+random: false
+workers: 1
+waitTime: 0s
+duration: 5s
+jobs:
+    - name: Some Job
+      steps:
+      - name: Some Step
+        action:
+          type: HttpRequest
+          httpHeaders:
+            key: 1
+          method: GET
+          url: %s`, TestServer.CreateURL("/people"))
+
+				summary, err := test.ExecutePlanFromData(plan, "--summary")
+
+				Expect(err).To(BeNil())
+
+				var runningTimeToParse = strings.Replace(summary.RunningTime, "s", "", -1)
+				runningTimeValue, _ := strconv.ParseFloat(runningTimeToParse, 64)
+				Expect(math.Floor(runningTimeValue)).To(Equal(float64(5)))
+			})
+
+			It("List Usage", func() {
+				list := []string{
+					fmt.Sprintf(`%s -X GET'`, TestServer.CreateURL("/people")),
+				}
+				summary, err := test.ExecuteList(list, "--summary", "--duration", "5s")
+				Expect(err).To(BeNil())
+
+				var runningTimeToParse = strings.Replace(summary.RunningTime, "s", "", -1)
+				runningTimeValue, _ := strconv.ParseFloat(runningTimeToParse, 64)
+				Expect(math.Floor(runningTimeValue)).To(Equal(float64(5)))
+			})
+		})
+	})
+
+	It("Halts execution if a payload input file is not found", func() {
 		list := []string{
-			fmt.Sprintf(`%s -X POST `, URLForTestServer("/error")),
-			fmt.Sprintf(`%s -X POST `, URLForTestServer("/success")),
-			fmt.Sprintf(`%s -X POST `, URLForTestServer("/error")),
-			fmt.Sprintf(`%s -X POST `, URLForTestServer("/success")),
-			fmt.Sprintf(`%s -X POST `, URLForTestServer("/error")),
-			fmt.Sprintf(`%s -X POST `, URLForTestServer("/success")),
+			fmt.Sprintf(`%s -X POST -d '{"name":"talula"}'`, URLForTestServer("/success")),
+			fmt.Sprintf(`%s -X POST -d @missing-file.json`, URLForTestServer("/success")),
 		}
 
-		TestServer.Use(rizo.HTTPResponseFactory(func(w http.ResponseWriter) {
-			w.WriteHeader(500)
-		})).For(rizo.RequestWithPath("/error"))
-
-		output, err := test.ExecuteList("./corcel", list, "--summary")
-		Expect(err).To(BeNil())
+		summary, err := test.ExecuteList(list, "--summary")
+		Expect(err).ToNot(BeNil())
 
-		Expect(string(output)).To(ContainSubstring("Summary"))
+		Expect(summary.Error).To(ContainSubstring("Request body file not found: missing-file.json"))
 	})
 
 	It("Error non-http url in the urls file causes a run time exception #21", func() {
```

Comment:
I'm guessing that since the move to GH issue 21 is not the same as it was on GitLab, might be worth just removing the `#21` as I think the test name describes the behaviour sufficiently.